packages/sources/anchorage/test-payload.json

```diff
@@ -0,0 +1,3 @@
+{
+ "requests": [{}]
```

Comment:
If we do this? Will the soak test be skipped? Or it will run with empty input and fail?

Reply:
No, soak tests will be skipped only if we add the EA to the blacklist. This means it will try to send a an empty request to an EA. The soak tests are failing now for this EA anyway, I think we need deployment files first. I am ok skipping the soak tests now, but don't want to add the EA to blacklist so that after the deployments we can configure it. Also i need proper input data for soak tests.

Reply:
You don't need deployment files, you will in proper input as well as setting up the env var in that secrets repo

Reply:
Yeah the latter ones are not ready yet, so I'm fine with this now